pom.xml

```diff
@@ -58,12 +58,12 @@ limitations under the License.
         <compat.module>hbase-hadoop2-compat</compat.module>
         <junit.version>4.12</junit.version>
         <mockito.version>1.10.19</mockito.version>
-        <grpc.version>1.6.0</grpc.version>
+        <grpc.version>1.6.1</grpc.version>
         <!--NOTE: 4.1.14.Final is the first version that support shading -->
         <netty.version>4.1.14.Final</netty.version>
         <netty-tcnative-boringssl-static.version>2.0.5.Final</netty-tcnative-boringssl-static.version>
         <protobuff-java.version>3.3.1</protobuff-java.version>
-        <protoc.version>3.3.0</protoc.version>
+        <protoc.version>3.4.0</protoc.version>
```

Comment:
protobuf is still 3.3.0 in 1.6.1
https://github.com/grpc/grpc-java/blob/v1.6.1/build.gradle#L96

It'll be bumped in 1.7